exporter/awsxrayexporter/awsxray_test.go

```diff
@@ -42,6 +45,23 @@ func TestTraceExport(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+// test when 2 span have invalid xray id and 2 span have valid xray id
+func TestMixedValidSpanTraceExport(t *testing.T) {
```

Comment:
We can probably remove this comment and just put the information in the test name?

```suggestion
func TestExtractingTracesWithWrongTraceIDFormat(t *testing.T) {
```